src/views/Alteration.vue

```diff
@@ -1,83 +1,105 @@
 <template>
   <section class="pb-10">
-    <!-- Profile View -->
-    <template v-if="!isSummaryMode">
-      <header>
-        <h1>Company Information</h1>
-      </header>
-
-      <section class="mt-6">
-        <p>You are legally obligated to keep your company information up to date. Necessary fees will be applied as
-          updates are made.</p>
-      </section>
-
-      <your-company class="mt-10" />
-
-      <current-directors class="mt-10" />
-
-      <share-structures class="mt-10" />
-
-      <articles class="mt-10" />
-
-<!--      Commented out until required-->
-<!--      <agreement-type class="mt-10" />-->
-<!--      <detail class="mt-10" />-->
-    </template>
-
-    <!-- Summary View -->
-    <template v-else>
-      <header>
-        <h1>Review and Certify</h1>
-      </header>
-      <section class="mt-6">
-        <p>Review and certify the changes you are about to make to your company. Certain changes require an Alteration
-          Notice which will incur a $100.00 Fee.</p>
-      </section>
-
-      <!-- FUTURE: set `pleaseValidate` when user clicks File and Pay -->
-      <alteration-summary
-        class="mt-10"
-        :validate="getAppValidate"
-        @haveChanges="onAlterationSummaryChanges()"
-      />
-
-      <documents-delivery
-       class="mt-10"
-       :validate="getAppValidate"
-        @valid="setDocumentOptionalEmailValidity($event)"
-      />
-
-      <certify-section
-       class="mt-10"
-       :validate="getAppValidate"
-      />
-
-      <!-- STAFF ONLY: Court Order and Plan of Arrangement -->
-      <template v-if="isRoleStaff">
-        <header class="mt-10">
-          <h2>3. Court Order and Plan of Arrangement</h2>
+    <!-- Company Information-->
+    <v-slide-x-transition hide-on-leave>
```

Comment:
Hide whitespace changes to see this diff more clearly. I didn't change very much actually.